src/hotspot/os_cpu/linux_x86/os_linux_x86.cpp

```diff
@@ -544,6 +544,22 @@ void os::print_context(outputStream *st, const void *context) {
   st->print(", ERR=" INTPTR_FORMAT, (intptr_t)uc->uc_mcontext.gregs[REG_ERR]);
   st->cr();
   st->print("  TRAPNO=" INTPTR_FORMAT, (intptr_t)uc->uc_mcontext.gregs[REG_TRAPNO]);
+#ifndef MUSL_LIBC
```

Comment:
Isn't it compiling on MUSL? `ucontext_t` and `mcontext_t` seem to be equal looking at https://git.musl-libc.org/cgit/musl/tree/arch/x86_64/bits/signal.h
Also the context comes from the kernel and should have the same layout I thought.
If it isn't working then I guess the changes in debug.cpp won't work for MUSL either ;)

Reply:
Let's give it a try. See Commit nr. 6.